src/App.js

```diff
@@ -1,5 +1,5 @@
 import "./App.css"
-import {Home, Products, Wishlist, Cart, Login, SignUp} from "./pages"
+import {Home, Products, Wishlist, Cart, Login, SignUp, SingleProduct} from "./pages"
```

Comment:
Follow a pattern where you import modules first and then import local modules this will increase the readability 
Ex: 

```
import React from 'react'

import "./App.css"

```

Reply:
ok, I will follow a pattern